<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <resources><include location="../../../res/extui/extui.qrc"/></resources><author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QG_LineAngleOptions</class>
  <widget class="QWidget" name="QG_LineAngleOptions">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>22</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>4</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>300</width>
        <height>22</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>400</width>
        <height>22</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Line Angle Options</string>
    </property>
    <layout class="QHBoxLayout">
      <item>
        <widget class="QLabel" name="lAngle">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text">
            <string>Angle:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leAngle">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="toolTip" stdset="0">
            <string>Line angle</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lLength">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text">
            <string>Length:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLineEdit" name="leLength">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="toolTip" stdset="0">
            <string>Length of line</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="lSnapPoint">
          <property name="text">
            <string>Snap Point:</string>
          </property>
          <property name="wordWrap">
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QComboBox" name="cbSnapPoint">
          <property name="font">
            <font/>
          </property>
          <item>
            <property name="text">
              <string>Start</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>Middle</string>
            </property>
          </item>
          <item>
            <property name="text">
              <string>End</string>
            </property>
          </item>
        </widget>
      </item>
      <item>
        <widget class="Line" name="sep1">
          <property name="sizePolicy">
            <sizepolicy>
              <hsizetype>0</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="frameShape">
            <enum>QFrame::VLine</enum>
          </property>
          <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="0"/>
  <connections>
    <connection>
      <sender>leAngle</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_LineAngleOptions</receiver>
      <slot>updateAngle(QString)</slot>
    </connection>
    <connection>
      <sender>leLength</sender>
      <signal>textChanged(QString)</signal>
      <receiver>QG_LineAngleOptions</receiver>
      <slot>updateLength(QString)</slot>
    </connection>
    <connection>
      <sender>cbSnapPoint</sender>
      <signal>activated(int)</signal>
      <receiver>QG_LineAngleOptions</receiver>
      <slot>updateSnapPoint(int)</slot>
    </connection>
  </connections>
</ui>
